cmd/minishift/cmd/root.go

```diff
@@ -179,3 +199,34 @@ func ensureConfigFileExists(configPath string) {
 		}
 	}
 }
+
+func performPostUpdateExecution() error {
+	fmt.Println("Binary has been recently updated.")
+	markerPath := filepath.Join(constants.Minipath, constants.UpdateMarkerFileName)
+	file, err := ioutil.ReadFile(markerPath)
+	if err != nil {
+		return err
+	}
+
+	var markerData UpdateMarker
+	var confirm = "y"
+
+	json.Unmarshal(file, &markerData)
+	if !markerData.Force {
+		fmt.Printf("Do you want to update the default addons? [y/N]: ")
+		fmt.Scanln(&confirm)
+	}
+
+	if strings.ToLower(confirm) == "y" {
```

Comment:
I am confused, didn't we say that the confirmation is in the update command? We want to keep root.go free from user interaction. Otherwise we break the ability to script Minishift commands.

I think we said that we ask for confirmation in the update command and store the answer into the update marker file.